src/js/App/Feedback/FeedbackModal.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { Button, Card, CardBody, CardTitle, Label, Modal, ModalVariant, Text, TextContent, TextVariants } from '@patternfly/react-core';
+import { ExternalLinkAltIcon, OutlinedCommentsIcon } from '@patternfly/react-icons';
+import feedbackIllo from '../../../../static/images/feedback_illo.svg';
+import Feedback from './Feedback';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import { toggleFeedbackModal } from '../../redux/actions';
+import FeedbackSuccess from './FeedbackSuccess';
+import './Feedback.scss';
+
+const FeedbackModal = ({ user }) => {
+  const usePendoFeedback = useSelector(({ chrome: { usePendoFeedback } }) => usePendoFeedback);
+  const isOpen = useSelector(({ chrome: { isFeedbackModalOpen } }) => isFeedbackModalOpen);
+  const dispatch = useDispatch();
+  const [modalPage, setModalPage] = useState('feedbackHome');
+  const env = window.insights.chrome.getEnvironment();
```

Comment:
You import and call the getEnv function directly. No need for the window roundtrip.